src/components/Lottie/index.tsx

```diff
@@ -103,6 +103,7 @@ function Lottie({source, webStyle, shouldLoadAfterInteractions, ...props}: Props
             // eslint-disable-next-line react/jsx-props-no-spreading
             {...props}
             source={animationFile}
+            key={`${hasNavigatedAway}`}
```

Comment:
Why are we using a template string? Can we just pass the variable?

Reply:
Thanks for clarifying. That looks a bit odd though, maybe we could have used a different key, but I won't block on that